content/en/docs/release-notes/release-notes-1.7.md

```diff
@@ -7,12 +7,117 @@ type: "docs"
 
 ## Breaking Changes (You **MUST** read this before you upgrade!)
 
+⚠ Following their deprecation in version 1.5, the cert-manager API versions v1alpha2, v1alpha3, and v1beta1 have been removed.
+You must ensure that all cert-manager custom resources are stored in etcd at version v1
+and that all cert-manager `CustomResourceDefinition`s have only v1 as the stored version.
+
+Since release 1.7, `cmctl` can automatically migrate any deprecated API resources.
+Please download `cmctl-v1.7.0-beta.0` and read [Removing Deprecated API Resources]
+for full instructions.
+
+[Removing Deprecated API Resources]: https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/
+
 ## Major Themes
 
+### Additional Certificate Output Formats
+
+`additionalOutputFormats` is a field on the Certificate `spec` that allows
+specifying additional supplementary formats of issued certificates and their
+private key. There are currently two supported additional output formats:
+`CombinedPEM` and `DER`. Both output formats can be specified on the same
+Certificate.
+Read [Additional Certificate Output Formats] for more details and
+thanks to [@seuf](https://github.com/seuf) for getting this across the line!
+
+[Additional Certificate Output Formats]: ../../usage/certificate/#additional-certificate-output-formats
+
+### Removal of Deprecated APIs
+
+In 1.7 the cert-manager API versions v1alpha2, v1alpha3, and v1beta1 have been removed from the custom resource definitions (CRDs).
+You will notice that the YAML manifest files are much smaller as a result.
+These APIs have been deprecated since 1.5.
+
+In this release we have added a new sub-command to the cert-manager CLI (`cmctl upgrade migrate-api-version`),
+which you SHOULD run BEFORE upgrading cert-manager to 1.7.
+Please read [Removing Deprecated API Resources] for full instructions.
```

Comment:
suggestion: I think I'd prefer to have Removal of Deprecated APIs before anything else under "Major Themes", it feels like the most important bit to me for people who might be reading these notes. It does stutter a bit following on from the breaking changes section, though. Totally not a blocker, just a personal preference thing.